File: 백준/Bronze/5543. 상근날드/상근날드.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int minBuger = 2001;
        int minDrink = 2001;

        for(int i = 0; i < 3; i++){
            int buger = Integer.parseInt(br.readLine().trim());
            if (buger < minBuger) minBuger = buger;
        }

        for(int i = 0; i < 2; i++){
            int drink = Integer.parseInt(br.readLine().trim());
            if (drink < minDrink) minDrink = drink;
        }

        br.close();
        System.out.println(minBuger + minDrink - 50);
    }
}

File: 백준/Bronze/5543. 상근날드/상근날드.py
import sys

input = sys.stdin.readline
print = sys.stdout.write

min_buger = 2001
min_drink = 2001

for _ in range(3):
    buger = int(input().strip())
    if buger < min_buger:
        min_buger = buger

for _ in range(2):
    drink = int(input().strip())
    if drink < min_drink:
        min_drink = drink

print(f"{min_buger + min_drink - 50}\n")